Description: Specify mode for an open() call to avoid FTFBS with glibc2.8
Author: Andrea Colangelo <warp10@ubuntu.com>

--- gbemol-0.3.2.orig/src/gbemol-utils.c
+++ gbemol-0.3.2/src/gbemol-utils.c
@@ -155,7 +155,7 @@ gboolean gbemol_cfg_create_file ()
 
 	path = g_strdup_printf ("%s/.gbemol.cfg", g_get_home_dir());
 
-	if ((fd = open (path, O_CREAT)) != -1)
+	if ((fd = open (path, O_CREAT, 0777)) != -1)
 	{
 		chmod (path, 0644);
 		close (fd);
